### PR TITLE
cachi2 wheels: documentation

### PR DESCRIPTION
**Pull request by slimreaper35**
# Maintainers will complete the following section

- [x] Commit messages are descriptive enough
- [x] Code coverage from testing does not decrease and new code is covered
- [x] Docs updated (if applicable)
- [x] Docs links in the code are still valid (if docs were updated)

**Note:** if the contribution is external (not from an organization member), the CI
pipeline will not run automatically. After verifying that the CI is safe to run:

- [approve GitHub Actions workflows][approve-gh-actions] by clicking a button
- approve the Red Hat Trusted App Pipeline container build by commenting `/ok-to-test`
  (as is the [standard for Pipelines as Code][pac-running-pipeline])

[approve-gh-actions]: https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks
[pac-running-pipeline]: https://pipelinesascode.com/docs/guide/running/#running-the-pipelinerun

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -210,7 +209,7 @@ make test-unit
 ```
 
 For finer control over which tests get executed, e.g. to run all tests in a specific file, activate
-the [virtualenv](#virtualenv) and run:
+the [virtualenv](Makefile) and run:
```
Comment (ben-alkov) [inline README.md:212]: This doesn't really have anything to do with activating the venv... that would be `./venv/bin/activate`

Comment (slimreaper35) [reply README.md]: Fixed. (It was just a bad link to virtualenv section that does not exist; it should be virtual-environment.)

File: docs/pip.md [hunk 2]
```diff
@@ -565,10 +586,13 @@ installation. If you do manage to make it work, please let us know.
...
 
-Some projects do not distribute sdists to PyPI. For example, [tensorflow](https://pypi.org/simple/tensorflow/) (as of
+Some projects do not distribute sdists to PyPI. For example, [tensorflow](https://pypi.org/project/tensorflow/2.11.0/#files) (as of
 version 2.11.0) distributes only wheels.
 
-Possible workaround: find the git repository for the project, get the source tarball for a release. In requirements.txt,
+Possible workarounds:
+
+* Enable pre-fetching wheels using `"allow_binary": "true"` in JSON input.
+* Find the git repository for the project, get the source tarball for a releaseIn requirements.txt,
```
Comment (ben-alkov) [inline docs/pip.md:595]:
```suggestion
* Find the git repository for the project, get the source tarball for a release. In requirements.txt,
```

Lost a period here, I think.

**Review by ben-alkov — APPROVED** [review]
Otherwise, LGTM.

File: docs/pip.md
Comment (taylormadore) [inline docs/pip.md:1]: Do we want to add/update the links at the top of this page to include the new "Distribution formats" section?

Comment (slimreaper35) [reply docs/pip.md:1]: Yes, I just forgot 😄

File: docs/pip.md [hunk 3]
```diff
@@ -358,11 +362,28 @@ Sdists are more difficult to install. Pip must first build a wheel from the sdis
 a [PEP 517](https://peps.python.org/pep-0517/) build system. To do that, pip has to install the build system and
 its dependencies (defined via [PEP 518](https://peps.python.org/pep-0518/)).
 
-Building from source gives you an important guarantee which using pre-built artifacts does not: what you installed
-matches the source code. This can be especially important for Python packages implemented in C or other compiled
-languages.
+Cachi2 (unlike the older Cachito) can download both wheels and sdists. The `allow_binary` option controls this behavior.
```
[the remark this answers is not in the document]
Comment (slimreaper35) [reply docs/pip.md:365]: I'm OK with either way, too, but Cahito is mentioned at multiple places in the [docs](https://github.com/search?q=repo%3Acontainerbuildsystem%2Fcachi2+cachito+language%3AMarkdown&type=code&l=Markdown)

**Review by taylormadore — APPROVED** [review]

**Pull request merged**